src/client/renderer/r_main.c

```diff
@@ -512,17 +518,78 @@ static void R_InitConfig(void) {
 
 	memset(&r_config, 0, sizeof(r_config));
 
+	// initialize GL pointers
+	gladLoadGL();
+
 	r_config.renderer_string = (const char *) glGetString(GL_RENDERER);
 	r_config.vendor_string = (const char *) glGetString(GL_VENDOR);
 	r_config.version_string = (const char *) glGetString(GL_VERSION);
-	r_config.extensions_string = (const char *) glGetString(GL_EXTENSIONS);
 
-	glGetIntegerv(GL_MAX_TEXTURE_UNITS, &r_config.max_texunits);
-	glGetIntegerv(GL_MAX_TEXTURE_IMAGE_UNITS, &r_config.max_teximage_units);
+	glGetIntegerv(GL_MAX_TEXTURE_IMAGE_UNITS, &r_config.max_texunits);
+
+	Com_Print("  Renderer:   ^2%s^7\n", r_config.renderer_string);
+	Com_Print("  Vendor:     ^2%s^7\n", r_config.vendor_string);
+	Com_Print("  Version:    ^2%s^7\n", r_config.version_string);
+
+	if (verbose->integer >= 1) {
+
+		GString *extension_string = g_string_new((const gchar *) glGetString(GL_EXTENSIONS));
+
+		// extension string can be gigantic, so let's pretty it up a bit.
+		GString *pretty_string = g_string_new(NULL);
+		const char *extensions_in = extension_string->str;
+		const char *extension_prefix = "  Extensions: ";
+		int num_waiting = 0;
+
+		while (true) {
+
+			// up here, we are assumed to be either at EOF or at a GL_ boundary.
+			// just be sure first.
+			while (*extensions_in && *extensions_in != 'G') {
+				++extensions_in;
+			}
+
+			// done, print what's left
+			if (!*extensions_in) {
+
```

Comment:
It looked even uglier just printing the blank string, because of wrapping. I wouldn't mind going back to just printing the raw extension string, but there's a bug relating to colors on huge strings that I'd need to fix as well.